### DISCUSSION TITLE
File Name from source to sink

### DISCUSSION THREAD

**Discussion by Manniac07** [vectordotdev/vector#19301] [Q&A] [answered]
Is it possible to pass the file name from the source include? to a label in the sink?
I know this is probably simple if its possible I just can't find where to do it. my sink is Prometheus Remote Write

**Comment by jszwedko** [comment]
Is this for the `file` source? If so, every event should have a `file` key on it indicating the file it came from.

**Comment by Manniac07** [comment]
@jszwedko do you know of any examples showing how to reference the file_key in a lua transform?

**Reply by jszwedko** [reply]
If `file_key` is set to `file`, for example, it would be `event.log.file`.

**Reply by Manniac07** [reply, marked as answer]
Thank you very much that worked
